swift_scality_backend/diskfile.py

```diff
@@ -528,9 +526,7 @@ def reader(self, keep_cache=False):
 
         :param keep_cache:
         """
-        dr = DiskFileReader(self._filesystem, self._name,
-                            int(self._metadata['Content-Length']),
-                            self._metadata['ETag'])
+        dr = DiskFileReader(self._filesystem, self._name)
```

Comment:
I'm somewhat surprised this still works, since the `DiskFileReader` constructor still takes 4/5 arguments?